onemask/ArchitectureStudy/app/src/main/java/com/android/studyfork/repository/UpbitDataSource.kt

```diff
@@ -0,0 +1,19 @@
+package com.android.studyfork.repository
+
+import com.android.studyfork.network.model.MarketResponse
+import com.android.studyfork.network.model.TickerResponse
+import io.reactivex.Single
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/**
+ * created by onemask
+ */
+interface UpbitDataSource {
+    @GET("market/all")
+    fun getMarketAll() : Single<List<MarketResponse>>
+
+    @GET("ticker")
+    fun getTikers(@Query("markets") markets : String? ): Single<List<TickerResponse>>
```

Comment:
메서드 이름이 `getTickers` 인게 맞을 것 같습니다
변수의 타입을 붙일때는 콜론(:) 을 이름 바로 뒤에 붙이는걸 권장하고 있습니다
```suggestion
    fun getTickers(@Query("markets") markets: String?): Single<List<TickerResponse>>
```

Reply:
오호 그렇군요 감사합니다 수정하였습니다.
ffad87e